### DISCUSSION TITLE
About the _k parameter

### DISCUSSION THREAD

**Discussion by MrChenhtlss** [rustdesk/rustdesk#5888] [Q&A]
I added the _k parameter when running hbbs. When building the client, I changed the key to the generated key value. When connecting, it prompted a key error.

I will be very grateful for your help！！！

**Comment by affarinha** [comment]
Are you sure you wrote it correctly?  -k _
There's a space between "k" and underscore.

**Reply by MrChenhtlss** [reply]
Thank you very much for your reply

It is a complete docs installation and running installation, as follows:

pm2 hbbs start -- -k  _

pm2 start hbbr

**Reply by MrChenhtlss** [reply]
The build modifications are as follows

[https://github.com/rustdesk/rustdesk/blob/master/libs/hbb_common/src/config.rs](url)
`pub const RS_PUB_KEY: &str = match option_env!("RS_PUB_KEY") {
    Some(key) if !key.is_empty() => key,
    _ => "Key generated by hbbs",
};`
